x-pack/plugins/security/server/lib/authorization/service.test.ts

```diff
@@ -61,34 +61,45 @@ test(`returns exposed services`, () => {
   const mockShieldClient = Symbol();
   getClient.mockReturnValue(mockShieldClient);
   const mockCheckPrivilegesWithRequest = Symbol();
-  checkPrivilegesWithRequestFactory.mockReturnValue(mockCheckPrivilegesWithRequest);
+  (checkPrivilegesWithRequestFactory as jest.Mock).mockReturnValue(mockCheckPrivilegesWithRequest);
```

Comment:
__optional nit:__ alternatively, to avoid these explicit `as jest.Mock` casts, we could organize mocks like this:
```ts
const mockCheckPrivilegesWithRequestFactory = jest.fn();
jest.mock('./check_privileges', () => ({
  checkPrivilegesWithRequestFactory: mockCheckPrivilegesWithRequestFactory,
}));

... no need to import .....

mockCheckPrivilegesWithRequestFactory.mockReturnValue(mockCheckPrivilegesWithRequest);
```

Reply:
Your suggestion is great, I was under the impression we weren't able to do this for some reason, perhaps I'm misremembering.